releases/release-5.0.4.md

```diff
@@ -0,0 +1,165 @@
+---
+title: TiDB 5.0.4 Release Notes
+---
+
+# TiDB 5.0.4 Release Notes
+
+Release Date: September 27, 2021
+
+TiDB version: 5.0.4
+
+## Compatibility Changes
+
++ TiDB
+
+    - Revert #19341 to avoid #24326 [#26258](https://github.com/pingcap/tidb/pull/26258)
+
+## Improvements
+
++ TiDB
+
+    - Trigger auto-analyze based on histogram row count [#26707](https://github.com/pingcap/tidb/pull/26707)
+
++ TiKV
+
+    - separate read write ready to reduce read latency [#10620](https://github.com/tikv/tikv/pull/10620)
+
```

Comment:
- Support changing CDC configs dynamically [#10685](https://github.com/tikv/tikv/pull/10685)
    - Reduce resolved ts message size to save network bandwidth. [#10678](https://github.com/tikv/tikv/pull/10678)
    - Drop log instead of blocking threads when slogger thread is overloaded and queue is filled up. [#10864](https://github.com/tikv/tikv/pull/10864)
    - TiKV coprocessor slow log will only consider time spent on processing the request. [#10864](https://github.com/tikv/tikv/pull/10864)
    - Make prewrite as idempotent as possible to reduce the chance of undetermined errors. [#10587](https://github.com/tikv/tikv/pull/10587)
    - Avoid false "GC can not work" alert under low write flow. [#10662](https://github.com/tikv/tikv/pull/10662)
    - Database restored from BR or Lightning Local-backend is now smaller, should be matching the original cluster size when backed up. [#10643](https://github.com/tikv/tikv/pull/10643)
    - Ensure panic output is flushed to the log [#10487](https://github.com/tikv/tikv/pull/10487)

Reply:
* PD 
  * Improved the performance of synchronizing Region information between PDs. [#3934](https://github.com/tikv/pd/pull/3934)
  * TiDB Dashboard: Add OIDC based SSO support [#3884](https://github.com/tikv/pd/pull/3884)